src/components/Drift/index.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
+import PropTypes from "prop-types";
+
+import { useChat } from '../../'
+import STATES from '../../utils/states'
+
+const styles = {
+  widgetContainer: {
+    zIndex: 2147483648,
+    position: 'fixed',
+    display: 'block',
+    bottom: '24px',
+    right: '24px',
+    height: '76px !important',
+    width: '76px !important',
+  },
+  container: {
+    WebkitFontSmoothing: 'antialiased',
+    fontSize: 16,
+    display: 'flex',
+    justifyContent: 'flex-end',
+    alignItems: 'flex-end',
+    padding: '.75rem',
+    flexDirection: 'column'
+  },
+  button: {
+    textAlign: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    fill: '#fff',
+    cursor: 'pointer',
+    height: 52,
+    width: 52,
+    borderRadius: '.3125rem',
+    boxShadow: '0 2px 6px 0 rgba(0,0,0,.4)',
+    overflow: 'hidden',
+    outline: 'none',
+    border: 'none'
+  },
+  iconWrapper: {
+    fill: 'inherit',
+    stroke: 'inherit',
+    width: '100%',
+    padding: 0,
+    lineHeight: 0
+  }
+}
+
+const Drift = ({ color, icon }) => {
+  const [state, loadChat] = useChat({ loadWhenIdle: true })
+
+  if (state === STATES.COMPLETE) return null
+  return (
+    <div style={styles.widgetContainer}>
+      <div style={styles.container}>
+        <div
+          onClick={() => loadChat({ open: true })}
+          onMouseEnter={() => loadChat({ open: false })}
+          style={{
+            backgroundColor: color,
+            ...styles.button
+          }}
+        >
+          <i 
+            style={{
+              ...styles.iconWrapper,
+              opacity: state === STATES.INITIAL ? 1 : 0.75
+            }}
+          >
+            {icon === "A" ? (
+              <svg
+                width="20"
+                height="20"
+                viewBox="0 0 20 20"
+                focusable="false"
+                aria-hidden="true"
+                style={{
+                  display: state === STATES.INITIAL ? "initial" : "none"
+                }}
+              >
+                <path
+                  d="M4.583 14.894l-3.256 3.78c-.7.813-1.26.598-1.25-.46a10689.413 10689.413 0 0 1 .035-4.775V4.816a3.89 3.89 0 0 1 3.88-3.89h12.064a3.885 3.885 0 0 1 3.882 3.89v6.185a3.89 3.89 0 0 1-3.882 3.89H4.583z"
+                  fill="rgb(255, 255, 255)"
+                  fill-rule="evenodd"
```

Comment:
This should be `fillRule`, the linter should pick this up on the other icons too.